docs/Developers/Review.rst

```diff
@@ -83,10 +83,13 @@ The review process for coala is as follows:
     .. note::
 
         Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
-
-4. If the commits are not linearly mergeable into master, rebase and go
+4. After the PR has been reviewed, don't comment on a review unless
```

Comment:
It feels like an odd place to put it here , because `ack`s in step 3 indicate the patch is ready to be merged.

I think this deserves a separate section before Manual Review Process , which can generally talk about best practises for review discussions.  In this case, the issue you are solving, the guidance is directed at the coder.

But we also have other guidance we should be adding aimed at reviewers.

@li-boxuan , we also have some guidance about not deleting / edit comments - is that here, or in a different document ?

Reply:
> we also have some guidance about not deleting / edit comments - is that here, or in a different document ?

@jayvdb It's in https://coala-api.netlify.com/developers/meta-review#meta-review-process

https://github.com/coala/coala/blob/6431b6e219831e2d33a668482e7b9e0e199b8e75/docs/Developers/Meta-review.rst#L79-L82

Reply:
@jayvdb so how should I proceed further for this PR?

Reply:
@jayvdb moved to separate section.